meta-luneos/recipes-webos-ose/qml-webos-framework/qml-webos-framework/0002-Setup-missing-env-variables.patch

```diff
@@ -7,31 +7,21 @@ Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>
 Upstream-Status: Pending
 
 ---
- tools/runner/main.cpp | 15 +++++++++++++++
- 1 file changed, 15 insertions(+)
+ tools/runner/main.cpp | 5 +++++
+ 1 file changed, 5 insertions(+)
 
 diff --git a/tools/runner/main.cpp b/tools/runner/main.cpp
 index 44009bf..bb2ae99 100644
 --- a/tools/runner/main.cpp
 +++ b/tools/runner/main.cpp
-@@ -130,6 +130,21 @@ int main(int argc, char *argv[])
+@@ -130,6 +130,11 @@ int main(int argc, char *argv[])
          qputenv("WEBOS_DEVICE_PIXEL_RATIO", QByteArray("auto"));
          qDebug("Set default WEBOS_DEVICE_PIXEL_RATIO as 'auto'");
      }
-+    if (qEnvironmentVariableIsEmpty("XDG_RUNTIME_DIR")) {
-+        // Set /tmp/luna-session" by default
-+        qputenv("XDG_RUNTIME_DIR", QByteArray("/tmp/luna-session"));
-+        qDebug("Set default XDG_RUNTIME_DIR as '/tmp/luna-session'");
-+    }
 +    if (qEnvironmentVariableIsEmpty("QT_QPA_PLATFORM")) {
 +        // Set "webos-wayland" by default
 +        qputenv("QT_QPA_PLATFORM", QByteArray("webos-wayland"));
```

Comment:
Should this be webos-wayland, or wayland-egl? we have an inconsistency it seems

Reply:
See above, I'm also confused here....

Reply:
webos-wayland should be the normal setting for apps, at least in my experience, otherwise they don't have the ability to know about the extensions like window type.